lerna-package-lock.json

```diff
@@ -3602,9 +3602,9 @@
 			"integrity": "sha512-KaoQ7w2MDH5OeRKVatL5yVOCFg+9wD6bLSLFh1/TV1EZM46l49iBqO7UVjUtPE6BIm0jvvOzJXULGVSpzokX3g=="
 		},
 		"@fluidframework/common-utils": {
-			"version": "0.32.0-31336",
-			"resolved": "https://registry.npmjs.org/@fluidframework/common-utils/-/common-utils-0.32.0-31336.tgz",
-			"integrity": "sha512-Hk7jT/L8GeQkFTripkYPkRsNKVedJUIzZxq0ESkhlegKRcFzN8YeLzO62CnPl2I+7Yhc/+0SZBGehAo8wyg0nA==",
+			"version": "0.32.1",
+			"resolved": "https://registry.npmjs.org/@fluidframework/common-utils/-/common-utils-0.32.1.tgz",
+			"integrity": "sha512-8m7nTGLyzQqjGX9qZmE1L65IfSxrcEo35MtCVmH0qfttfVqEFghxXAgtmrcfvKkg6NUDTmb13hdyNx0hZlky7w==",
```

Comment:
hmm is this right?

Reply:
yeah i already ran the release for 0.32.1 so it should be picking it up in the lock file